Frontends/SPIRV/SpvToTop.cpp

```diff
@@ -911,7 +913,9 @@ gla::EMdInputOutput SpvToTopTranslator::getMdQualifier(spv::Id resultId) const
     case gla::EmbPointSize:   mdQualifier = gla::EMioPointSize;      break;
     case gla::EmbClipVertex:  mdQualifier = gla::EMioClipVertex;     break;
     case gla::EmbInstanceId:  mdQualifier = gla::EMioInstanceId;     break;
+    case gla::EmbInstanceIndex: mdQualifier = gla::EMioPipeIn;         break;
```

Comment:
Why doesn't the default catch this?  Note that almost all built-in variables are missing in this list, so curious why the new ones show up as special cases.